plugin/vim-copilot-ignore.vim

```diff
@@ -13,28 +13,22 @@ let g:loaded_copilotignore = 1
 function! s:ReadIgnorePatterns(filepath) abort
   let l:patterns = []
   if filereadable(a:filepath)
+    let filefolder = fnamemodify(a:filepath, ':p:h')
+    let local = filefolder ==# $HOME ? v:false : v:true
     let l:lines = readfile(a:filepath)
     for l:line in l:lines
-      call add(l:patterns, substitute(l:line, '^\s*\(.\{-}\)\s*$', '\1', ''))
+      call add(l:patterns,
+            \ (local ? filefolder..(!exists("+shellslash") || &shellslash ? '/' : '\') : '')..trim(l:line))
```

Comment:
I think this is for more cross-platform file/path compatibility?

Reply:
`/` is usually used under Unix, say Linux or Macos, whereas `\` is for Microsoft Windows, except if `&shellslash` is set.